### ISSUE TITLE
Highlight images without alt text

### ISSUE THREAD

**Issue by abraham** [gdg-x/hoverboard#2252]
All images should have alt text. Any that do not should be visually highlighted so it's noticeable by site maintainers.

https://simonblackbourn.net/blog/highlight-images-missing-alt-attribute
